r2-lcp/src/main/java/org/readium/r2/lcp/service/DeviceRepository.kt

```diff
@@ -9,9 +9,23 @@
 
 package org.readium.r2.lcp.service
 
+import org.readium.r2.lcp.LcpException
 import org.readium.r2.lcp.license.model.LicenseDocument
+import org.readium.r2.lcp.persistence.LcpDao
 
-internal interface DeviceRepository {
-    fun isDeviceRegistered(license: LicenseDocument) : Boolean
-    fun registerDevice(license: LicenseDocument)
+class DeviceRepository(private val lcpDao: LcpDao) {
```

Comment:
We should keep the repositories private.

```suggestion
internal class DeviceRepository(private val lcpDao: LcpDao) {
```